manifest/rhel_on_kubernetes/cluster-network-03-nsxerrors.nsx.vmware.com.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
How about adding subdir /rhel and /ubuntu under manifest/kubernetes/

Reply:
done